schema.graphql

```diff
@@ -157,7 +157,7 @@ type ConsignmentStatus {
 input ConsignmentStatusInput {
   consignmentId: UUID!
   statusType: String!
-  statusValue: String!
+  statusValue: String
```

Comment:
I don't think so because I left it out in the RouteSpec tests [here](https://github.com/nationalarchives/tdr-consignment-api/pull/449/files#diff-9c6f7642b0572956c29857db1de71fe336af0c726f5af176d97afb7bb93fe2db). I think if you pass a value, Sangria  will wrap it in a `Some` and if you don't, it will just make it a `None`

Reply:
Ah OK, for once something useful. So yeah, makes sense to make it optional.